.github/workflows/macos.yml

```diff
@@ -137,8 +137,9 @@ jobs:
           echo -e "  QMAKE_MACOSX_DEPLOYMENT_TARGET = ${{ env.QMAKE_MACOSX_DEPLOYMENT_TARGET }}"  >> ./input/app/config.pri
           echo -e "  QGIS_QUICK_DATA_PATH = ${{ github.workspace }}/input/app/android/assets/qgis-data"  >> ./input/app/config.pri
           echo -e "  QMAKE_CXXFLAGS += --coverage"  >> ./input/app/config.pri
-          echo -e "  QMAKE_LFLAGS += --coverage "  >> ./input/app/config.pri
-          echo -e "  QMAKE_CXX = ccache \$\$QMAKE_CXX "  >> ./input/app/config.pri
+          echo -e "  QMAKE_LFLAGS += --coverage"  >> ./input/app/config.pri
+          echo -e "  QMAKE_CXX = ccache \$\$QMAKE_CXX"  >> ./input/app/config.pri
+          echo -e "  CONFIG += sdk_no_version_check"  >> ./input/app/config.pri
```

Comment:
Do we want to include this one? `sdk_check` only prints a warning, no? I wonder if that would not hide some errors in the future

Reply:
it is Mac and we do not release it .... and this is only CI, you still have warning on your local build...

Reply:
feel free to remove on master if you want, I am easy either way